mne/preprocessing/_csd.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2003-2010 Jürgen Kayser <rjk23@columbia.edu>
+# Copyright 2017 Federico Raimondo <federaimondo@gmail.com> and
+#                Denis A. Engemann <dengemann@gmail.com>
+#
+#
+# The original CSD Toolbox can be find at
+# http://psychophysiology.cpmc.columbia.edu/Software/CSDtoolbox/
+
+# Authors: Denis A. Engeman <denis.engemann@gmail.com>
+#          Alex Rockhill <aprockhill206@gmail.com>
+#
+# License: BSD (3-clause)
```

Comment:
Something like:

Relicensed under BSD and adapted with permission from authors of original GPL code ...